README.md

```diff
@@ -63,6 +63,24 @@ Data Collector
 --------------
 Drop `website/UPDATE.PHP` and `website/internals.php` somewhere, and rename `UPDATE.PHP` to something secret. Make sure you don't have directory listings enabled.
 
+Slave DNS Config
+----------------
+Before running the benchmarks, add these lines to your `/etc/hosts` file and flush DNS cache. These host addresses are used by `benchmarks_remote.py`. This configuration is needed for all the slaves.
+
```

Comment:
that use the remote or local benchmarks. Only the shell benchmarks don't need it.